apis/v1alpha2/policy_types.go

```diff
@@ -16,6 +16,18 @@ limitations under the License.
 
 package v1alpha2
 
+const (
+	// PolicyLabelKey is the label whose presence identifies a CRD as a policy
+	// CRD. The value of the label SHOULD be one of the following:
+	//  - A label value of "inherited" indicates that this Policy is inheritable.
```

Comment:
I've gotten this wrong before (I think I'm also responsible for this time), but in general Kubernetes prefers for values of annotations to be capitalized. I'd recommend only specifying the upper case values in the case but maybe we can still have a case-insensitive match in tools like gwctl.

```suggestion
	//  - A label value of "Inherited" indicates that this Policy is inheritable.
```

Reply:
Sure that sounds good. Changed.